### PR TITLE
Vignettes update

### PR DESCRIPTION
**Pull request by PavelBal**
I have splitted the vignettes into separate files according to the topic.

-Added information about individual and populations
-Added information about various formula types.

### CODE REVIEW

**Comment by PavelBal** [comment]
OK vignettes are failing because PK-Sim is not found... For "create individual" and "create population" cvignettes.

**Comment by msevestre** [comment]
we'll have to try to install PK-Sim during the build process then

**Comment by msevestre** [comment]
Working on this. I removed the section on initPKSim which was trying to initialize pksim with a path that is not valid

File: vignettes/unit-conversion.Rmd [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+The values of a certain dimension can be presented in different units - for examlpe, *l* or *ml* for the dimension *Volume*, or *mol* and *µmol* for the dimension *Amount*. The list of all available units is a method of an entity:
+
+```{r $allUnits}
+# Dimension of the parameter
+livParam$dimension
+# Units of the parameter
+livParam$allUnits
+```
+
+Internally, `OSPS` works with the **base units**, and all the values that are shown or passed to functions are in base units by default.
```
Comment (msevestre) [inline vignettes/unit-conversion.Rmd:45]: Please rename OSPS to OSPSuite

**Comment by msevestre** [comment]
yeah works. That means that we can now add tests for PKSim as well :)

**Pull request merged**